### DISCUSSION TITLE
Permissions on /opt/postal/config

### DISCUSSION THREAD

**Discussion by brywhi** [postalserver/postal#1776] [Help with using Postal] [answered]
Hello, we installed postal as instructed on https://docs.postalserver.io/install/installation

One issue we're having- I have to chmod 755 /opt/postal/config each time we restart the server/update it, otherwise we get logs about the containers not being able to access the config directory. It seems to go back to 700 each time we update. Should I try to make the 755 permanent, or did I screw something up in the installation?

**Comment by astorrer** [comment]
From Dockerfile.yml
# Setup an application
RUN useradd -r -d /opt/postal -m -s /bin/bash -u 999 postal
USER postal
RUN mkdir -p /opt/postal/app /opt/postal/config
WORKDIR /opt/postal/app

I wonder if it's something related to the docker containers.

**Comment by willpower232** [comment]
Are you changing permissions of the config directory on the host or inside the container?

**Comment by brywhi** [comment]
Directly on the host.

-BW

**Reply by willpower232** [reply]
That is weird, the permissions on the host shouldn't randomly change unless you have something else going on. Is there some security process or something resetting the permissions?

**Reply by brywhi** [reply]
I figured this one out. We had created a cron job on the host to automatically copy certs from the caddy-data directory into the config directory for the SMTP server to use, and that cron job breaking the permissions. I fixed the cron job and now we are good to go!

**Reply by willpower232** [reply]
ah excellent, we have similar cron jobs so will bear that in mind

**Comment by brywhi** [comment, marked as answer]
Not a problem with Postalserver, this was something we did to ourselves with a poorly written cron task.
